res/css/components/views/elements/_AppPermission.pcss

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+Copyright 2019 - 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_AppPermission {
+    > div {
+        margin-bottom: 12px;
+    }
+
+    h4 {
+        margin: 0;
+        padding: 0;
+    }
+
+    .mx_AppPermission_smallText {
+        font-size: $font-12px;
+    }
+
+    .mx_AppPermission_bolder {
+        font-weight: var(--font-semi-bold);
+    }
+
+    .mx_AppPermission_helpIcon {
+        margin-top: 1px;
+        margin-right: 2px;
+        width: 10px;
+        height: 10px;
+        display: inline-block;
+
+        &::before {
+            display: inline-block;
+            background-color: $accent;
+            mask-repeat: no-repeat;
+            mask-size: 12px;
+            width: 12px;
+            height: 12px;
+            mask-position: center;
+            content: "";
+            vertical-align: middle;
+            mask-image: url("$(res)/img/feather-customised/help-circle.svg");
+        }
+    }
+
+    .mx_AppPermission_tooltip {
```

Comment:
This style block has become obsolete and not used, and it should be fine to just remove it (https://github.com/vector-im/element-web/issues/25354). I'll revisit later.